LSP.sublime-settings

```diff
@@ -130,6 +130,30 @@
       "scopes": ["source.julia"],
       "syntaxes": ["Packages/Julia/Julia.sublime-syntax"],
       "settings": {"runlinter": true}
+    },
+    "polymer-ide":
+    {
+      "command": ["polymer-editor-service"],
+      "scopes": [
+        "text.html.basic",
+        "text.html",
+        "source.html",
+        "source.js",
+        "source.css"
+      ],
+      "syntaxes": [
+        "Packages/HTML/HTML.sublime-syntax",
+        "Packages/CSS/CSS.sublime-syntax",
+        "Packages/JavaScript/JavaScript.sublime-syntax",
+        "Packages/JavaScript/JSON.sublime-syntax"
+      ],
+      "languageId": "polymer",
```

Comment:
I guess it's hard to pick a languageId that covers all the document types, but we use this value to select a syntax highlighter for Signature Help popups. If you eg. only have Signature Help for Javascript, putting `javascript` here might be a better choice.

Reply:
Ah, interesting. We don't have signature help currently, but we could inside of Polymer databinding expressions, in which case we'd want javascript. Updated.

Reply:
On a related note, looks like in the latest version of the protocol a client can declare support for markdown in signature help and completion items documentation.